components/mdx/BlockQuote.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import styles from '../../scss/blockQuote.module.scss'
+const BlockQuote: React.FC<{}> = ({ children }) => {
```

Comment:
Don't need the empty generics.

```suggestion
const BlockQuote: React.FC = ({ children }) => {
```